### ISSUE TITLE
Please distinguish between I and l better

### ISSUE THREAD

**Issue by paul4ig** [google/roboto#226]
I absolutely hate Roboto as a user interface font as the I (uppercase i) and l (lowercase L) are not distinct enough. They can also get confused with vertical bars | or | as well as numerous other unicode characters. Think of the $billions wasted in global human effort due to all this confusion due to pathetic form-over-function font design that dominates. If you want to create the ultimate clear and functional font then please be doubly clear by adding bars to the I (uppercase i) and also add a small tail to the l (lowercase L).

**Comment by marekjez86** [comment]
sample text

<html lang="en">
<p><span style="font-size:24.0px">Ill behaviour is caused by an AI called Al III commissioned by the little men in the Illogical Illuminati</span></p>
<p><span style="font-size:24.0px">aR65Hus0lfG3</span></p>
<p><span style="font-size:24.0px">Illinois</span></p>
</html>
en-821-sentence-Roboto-Regular
en-821-sentence-roboto-regular
en-821-sentence-Roboto-Italic
en-821-sentence-roboto-italic
en-821-sentence-Roboto-Black
en-821-sentence-roboto-black

**Comment by marekjez86** [comment]
sample test text for the above request
```html
<html lang="en">
<p><span style="font-size:24.0px">Ill behaviour is caused by an AI called Al III commissioned by the little men in the Illogical Illuminati</span></p>
<p><span style="font-size:24.0px">aR65Hus0lfG3</span></p>
<p><span style="font-size:24.0px">Illinois</span></p>
</html>
```

en-821-sentence-Roboto-Regular
![en-821-sentence-roboto-regular](https://cloud.githubusercontent.com/assets/13322623/21690590/c99aa32c-d329-11e6-8c05-5837edd3d2d7.jpeg)
en-821-sentence-Roboto-Italic
![en-821-sentence-roboto-italic](https://cloud.githubusercontent.com/assets/13322623/21690604/d6af5fd0-d329-11e6-8c64-848f12a99709.jpeg)
en-821-sentence-Roboto-Black
![en-821-sentence-roboto-black](https://cloud.githubusercontent.com/assets/13322623/21690620/e32b0cc8-d329-11e6-9464-1dfc59411f5d.jpeg)

**Comment by fitojb** [comment]
I wonder how will you concoct this since Roboto is a grotesque design; it just won't fit. Lowercase ells do not necessarily _need_ to have tails to be distinguishable; the varying cap height of lowercase and uppercase helps on that, and let's not forget about context, which also helps disambiguate.

**Comment by paul4ig** [comment]
@fitojb of course it will fit. Just add a subtle tail like in Source Sans Pro (though this is still a flawed font as still uses a simple vertical bar for the I). This is what annoys me about many arty font designers -- too concerned about fitting their design into some sort of arbitrary fashion category rather than ensuring the prime design purpose of optimum **communication**. If Google wants to be truly innovative, their priority should be a standard UI font that is absolutely clear-cut in unambiguity. The average person reads many different fonts a day, and having the reader to be aware of the context of the individual font just isn't good enough. To have absolutely no unnecessary cognitive overhead the I needs to be unambiguous as well as an l -- neither should be a vertical bar.

**Comment by Mancerrss** [comment]
Yes make this happen. This is unacceptable to such a humanist focused design font like Roboto. Just look how Helvetica, San Francisco and Segoe pulled this off better

**Comment by howdev** [comment]
Is basically a flaw design. Several other fonts have this same problem, Nimbus Sans, Open Sans, Helvtica Neue, Google Sans etc.  Noto also by Google do not have this issue.

**Comment by howdev** [comment]
The best Open Source font I found is KDE Oxygen or the new Oxygen Sans fonts. They have nice shape and don't have the capital I and lower case, vertical bar issue.
